ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -42,8 +38,21 @@ public class ControlTokenSet<T: TokenSetKey>: ObservableObject {
         valueOverrides?[token] = nil
     }
 
-    public var globalTokens: GlobalTokens { fluentTheme.globalTokens }
-    public var aliasTokens: AliasTokens { fluentTheme.aliasTokens }
+    /// Convenience method to replace all overrides with a new set of values.
+    ///
+    /// Any value present in `overrideTokens` will be set onto this control. All other values will be
+    /// removed from this control. If overrideTokens is `nil`, then all current overrides will be removed.
+    ///
+    /// - Parameter overrideTokens: The set of tokens to set as custom, or `nil` to remove all overrides.
+    public func replaceAllOverrides(with overrideTokens: [T: ControlTokenValue]?) {
```

Comment:
Does this reset theme overrides? If it does, should we update the comment to make that more explicit?

Reply:
Looks like no, this shouldn't really touch theme values at all. Should be fine as is, then.